cmd/gotk/reconcile_helmrelease.go

```diff
@@ -144,7 +144,7 @@ func isHelmReleaseReady(ctx context.Context, kubeClient client.Client, name, nam
 		}
 
 		for _, condition := range helmRelease.Status.Conditions {
-			if condition.Type == helmv2.ReadyCondition {
+			if condition.Type == meta.ReadyCondition {
```

Comment:
This should take the new `LastHandledReconcilateAt` status field into account, and could make use of `meta.GetCondition()`.